devicetypes/smartthings/ikea-button.src/ikea-button.groovy

```diff
@@ -269,9 +270,12 @@ def parse(String description) {
 			} else if (descMap.clusterInt == CLUSTER_SCENES ||
 					descMap.clusterInt == zigbee.ONOFF_CLUSTER ||
 					descMap.clusterInt == zigbee.LEVEL_CONTROL_CLUSTER ||
-					descMap.clusterInt == CLUSTER_WINDOW_COVERING) {
+					descMap.clusterInt == CLUSTER_WINDOW_COVERING && (!isThirdReality())) {
 				event = getButtonEvent(descMap)
-			}
+			} else if ((descMap.clusterInt == zigbee.ONOFF_CLUSTER ||
+			    descMap.clusterInt == 0x0012 )&&(isThirdReality())) {
+				    event = getButtonEvent(descMap)
+				}
```

Comment:
this separate if statement is not required after the refactoring to move all the code into `getButtonEvent`